src/client/static/images/chat.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48">
```

Comment:
Is this file still used anywhere?

Reply:
Nice catch @brunadafonseca! I actually liked the chat icon though :-/ Now it feels like a floating button which is a bit out of place. I do like the spinner for `.button--pending`, but maybe we safe that for elsewhere?